po/es.po

```diff
@@ -6,20 +6,20 @@ msgstr ""
 "PO-Revision-Date: 2021-09-14 15:25\n"
 "Last-Translator: \n"
 "Language-Team: Spanish\n"
-"Language: es_ES\n"
+"Language: es\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 "X-Crowdin-Project: clapper\n"
 "X-Crowdin-Project-ID: 473374\n"
-"X-Crowdin-Language: es-ES\n"
+"X-Crowdin-Language: es\n"
```

Comment:
Both this and above "Language" is autogenerated with Crowdin. If this requires changing, I would need to change the configuration there instead. Why are you trying to change this? As far as I see, other GNOME projects also use `es_ES`. :thinking: